### ISSUE TITLE
Error when compiling the example on Windows

### ISSUE THREAD

**Issue by Laaen** [karlseguin/http.zig#57]
Hello !

I get these errors when trying to compile the example in Windows

```
PS C:\Users\laaen\Desktop\Zig\Zig> ..\zig-windows-x86_64-0.13.0\zig.exe build                                                              
install
└─ install Zig
   └─ zig build-exe Zig Debug native 3 errors
C:\Users\laaen\AppData\Local\zig\p\1220eab1411af0e3217203d6891584cbcda8709f0bacec2aa2b1b55a1a1703559edf\src\httpz.zig:377:69: error: expected type '*anyopaque', found 'comptime_int'
                self._signalers[0] = .{.lock = .{}, .pipe = [2]fd_t{0, socket}};
                                                                    ^
referenced by:
    start: src\simple.zig:29:15
    main: src\main.zig:19:48
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
C:\Users\laaen\AppData\Local\zig\p\1220eab1411af0e3217203d6891584cbcda8709f0bacec2aa2b1b55a1a1703559edf\src\httpz.zig:377:69: error: expected type '*anyopaque', found 'comptime_int'
                self._signalers[0] = .{.lock = .{}, .pipe = [2]fd_t{0, socket}};
                                                                    ^
C:\Users\laaen\AppData\Local\zig\p\1220eab1411af0e3217203d6891584cbcda8709f0bacec2aa2b1b55a1a1703559edf\src\httpz.zig:377:69: error: expected type '*anyopaque', found 'comptime_int'
                self._signalers[0] = .{.lock = .{}, .pipe = [2]fd_t{0, socket}};
                                                                    ^
```

I'm using the zig-0.13 branch with  Zig version 0.13.0 on Windows 11

I don't know if it's something on my side, as it's my first experience with Zig, but, I followed your instructions (zig fetch and then add the two lines in the build.zig file)

Thanks in advance for your help !

**Comment by karlseguin** [comment]
I'm sorry about makign this your first experience with Zig.

I believe the issue is now fixed in the 0.13 branch (and also in the master branch).

In the branch that I'm currently working on (nonblocking_ws), there's no a github action which compiles to windows, so hopefully these types of issues will go away.

**Comment by Laaen** [comment]
It works perfectly fine now, thanks ! :D
